Linked Accounts/Link-Accounts.ps1

```diff
@@ -613,98 +528,94 @@ Write-LogMessage -Type Info -MSG "Getting PVWA Credentials to start Linking acco
 $caption = "Account Link Utility"
 $msg = "Enter your User name and Password";
 $creds = $Host.UI.PromptForCredential($caption, $msg, "", "")
-if ($null -ne $creds)
-{
-	if ($AuthType -eq "radius" -and ![string]::IsNullOrEmpty($OTP))
-	{
+if ($null -ne $creds) {
+	if ($AuthType -eq "radius" -and ![string]::IsNullOrEmpty($OTP)) {
 		$g_LogonHeader = $(Get-LogonHeader -Credentials $creds -RadiusOTP $OTP)
-	}
-	else
-	{
+	} else {
 		$g_LogonHeader = $(Get-LogonHeader -Credentials $creds )
 	}
-}
-else
-{
+} else {
 	Write-LogMessage -Type Error -MSG "No Credentials were entered" -Footer
 	exit
 }
 #endregion
 
+If (Compare-Version("11.7")) {$extraPass = "extraPasswordIndex"} else {$extraPass = "extraPasswordID"}
```

Comment:
@bab29 from this If I understand that there was a change in the API parameter name
so this means that we need to check a minimum version (that has the LinkAccounts API URL) and a change version (from which we had the parameter change)
I don't think a simple If/Else is enough here

WDYT?